androidApp/build.gradle.kts

```diff
@@ -53,6 +53,9 @@ android {
         implementation(libs.androidx.compose.foundation)
```

Comment:
What do you think about to create another module for `in-app update`? Like `core/google-play`?

Reply:
Unfortunately, Like I said in issue #92 discussion, I got some Gradle errors about something called tool chain and I can't build the project. So there's no `syntax highlighting` or `automatic import` and other IDE features.
I think it's because I opened the project in a Windows machine, but KMM projects require iOS.

Likewise, I don't want to bother you by asking questions about a Gradle problem which I probably can't fix.

I'm really sorry about this :(

Reply:
@alibardide5124 `KMM` works in **Windows** Machine,only there are some limitations. In **Windows** machine, you can only write Android and shared module code. So, you can build it easily. 

Could you please share me the your `Gradle` errors? Did you update project with last updates?

Reply:
Can I run it on Windows? Well, that's very good. Everywhere I searched, they said it needs mac. I'll be glad to help.

This was the Gradle error:  (I fixed it)

```
A problem occurred configuring project ':core:admob'.
> Could not determine the dependencies of null.
   > Could not resolve all dependencies for configuration ':core:admob:classpath'.
      > Failed to calculate the value of task ':build-logic:convention:compileJava' property 'javaCompiler'.
         > No matching toolchains found for requested specification: {languageVersion=11, vendor=any, implementation=vendor-specific}.
            > No locally installed toolchains match (see https://docs.gradle.org/8.0/userguide/toolchains.html#sec:auto_detection) and toolchain download repositories have not been configured (see https://docs.gradle.org/8.0/userguide/toolchains.html#sub:download_repositories).
```

I spent some time the other day, but I couldn't fix it. I thought maybe there is some tool in mac that windows doesn't have (Since it mentiond toolchain).
Funny, it turns out it needed `Java 11`. I had `Java 17` and `Java 19` but it needs exactly 11.

So I fixed it, but... Here is another error:
```
Execution failed for task ':shared:generateDummyFramework'.
> java.io.FileNotFoundException: C:\Users\Phoenix\AndroidStudioProjects\anypass-kmp\shared\build\cocoapods\framework\shared.framework\anypass:shared.bundle\dummyFile (The filename, directory name, or volume label syntax is incorrect)
```

I check it out and there's no folder named `cocoapods` in `shared\build` folder

Reply:
So just to follow project architecture:
I should make `core/google-play/in-app-update`.
And then in `kotlin` folder make a file named `InAppUpdate.android.kt` and implement logic there.

Is this correct?